v3.1/Dockerfile

```diff
@@ -1,30 +1,77 @@
 FROM owasp/modsecurity-crs:v3.1
 
-ENV PARANOIA_LEVEL=1 \
+ENV APACHE_RUN_USER=www-data \
+    APACHE_RUN_GROUP=root \
+    SERVER_ADMIN=root@localhost \
+    SERVER_NAME=localhost \
     PORT=8080 \
-    HTTPD_SERVER_LIMIT=16 \
     BACKEND=http://localhost:8080 \
-    REQ_BODY_LIMIT=100000000 \
-    REQ_BODY_NOFILES_LIMIT=5242880 \
+    APACHE_TIMEOUT=5 \
+    APACHE_LOGLEVEL=notice \
+    APACHE_ERRORLOG=/dev/stderr \
+    APACHE_ACCESSLOG="/dev/stdout extendedjson" \
+    APACHE_PERFLOG="/dev/stdout perflogjson env=write_perflog" \
+    MODSEC_AUDIT_LOG=/var/log/modsecurity/audit.log \
+    MODSEC_AUDIT_STORAGE=/var/log/modsecurity/audit/ \
+    MODSEC_DEBUG_LOG=/dev/null \
+    MODSEC_DEBUG_LOGLEVEL=0 \
+    MODSEC_DATA_DIR=/tmp/modsecurity \
```

Comment:
Every of the three directories (`DATA`, `TMP`, `UPLOAD`) should have an own subdirectory in `/tmp/modsecurity`.